x-pack/plugins/apm/server/lib/helpers/setup_request.ts

```diff
@@ -42,17 +42,14 @@ export interface Setup {
   ml?: ReturnType<typeof getMlSetup>;
   config: APMConfig;
   indices: ApmIndicesConfig;
+  uiFiltersES: ESFilter[];
```

Comment:
`uiFiltersES` is the most awkward name - and I can say that because I picked it...
Generalising it would probably be better:

```suggestion
  esFilter: ESFilter[];
```

But that's probably out of scope for this PR